src/applications/personalization/profile-2/sass/profile-2.scss

```diff
@@ -2,6 +2,9 @@
 @import "../../../../platform/forms/sass/m-schemaform";
```

Comment:
can this import be switched to 'platform/forms/sass/m-schemaform' ?  or does sass not play well with those imports?

Reply:
sadly no, Sass don't play that.